### ISSUE TITLE
v2.1.1 support

### ISSUE THREAD

**Issue by abenrob** [open-xml-templating/docxtemplater-image-module#58]
Hello.
Is there planned support for docxtemplater 2.1.1?
Some of the memory allocation issues that 2.1.1 handle are important, but images only work with 1.x.
Thanks.

**Comment by edi9999** [comment]
docxtemplater-image-module#v1 is compatible with docxtemplater#v2

**Comment by abenrob** [comment]
Ok. That's good! Let me rephrase the quesition then: 
What is the method for implementing docxtemplater-image-module#v1 with docxtemplater#v2?
I am running like this:
```
var opts = {};
opts.centered = false;
opts.getImage = function (tagValue, tagName) {
  var buf;
  if (typeof tagValue === 'string') {
    buf = new Buffer(imgobj[tagValue].img, 'base64').toString('binary');
  } else {
    buf = null;
  }
  return buf;
};

opts.getSize = function (img, tagValue, tagName) {
  return imgobj[tagValue].dims;
};

var imageModule = new ImageModule(opts);
var docData = {//my complex data};
var template = fs.readFileSync(templatePath, 'binary');
var doc = new Docxtemplater(template);
doc.attachModule(imageModule);
doc.setData(docData);
doc.render();
var buf = doc.getZip().generate({ type: 'nodebuffer' });
```

in Doc: {%imgId}
The opts.getImage function is not called at all; the rest of the doc works fine, but with no images.

**Comment by edi9999** [comment]
I think most probably your data doesnt contain imgId field

**Comment by abenrob** [comment]
tout à fait. Thanks for fielding ridiculous issues like mine!

**Comment by edi9999** [comment]
No problem !
